templates/blog/product-cards/_ai-solutions.html

```diff
@@ -0,0 +1,14 @@
+<div class="p-card js-product-card u-hide">
+    <h3>
+        <a href="https://canonical.com/solutions/ai">Enterprise AI, simplified</a>
+    </h3>
+    <img class="p-card__thumbnail"
+         src="https://assets.ubuntu.com/v1/e549fdc0-Canonical-AI-ML-Illustrations-v5-03.png"
+         alt="canonical ai ml illustration" />
```

Comment:
It might not require an alt="" here. Please refer the doc I had pinned in one of comments.